test/storage/recovery_test.cpp

```diff
@@ -244,6 +251,30 @@ class RecoveryTests : public TerrierTest {
         recovery_txn_manager_->Commit(recovery_txn, transaction::TransactionUtil::EmptyCallback, nullptr);
       }
     }
+
+    if (enable_metric) {
+      // Make sure everything is finished
+      std::this_thread::sleep_for(std::chrono::seconds(1));
+
+      // Metrics tests
+      const auto aggregated_data = reinterpret_cast<metrics::RecoveryMetricRawData *>(
+          recovery_manager.metrics_manager_->AggregatedMetrics()
+              .at(static_cast<uint8_t>(metrics::MetricsComponent::RECOVERY))
+              .get());
+      EXPECT_NE(aggregated_data, nullptr);
+
+      // Calculate total number of txns
+      uint64_t total_num_txns = 0;
+      uint64_t total_num_bytes = 0;
+      for (auto &data : aggregated_data->recovery_data_) {
+        total_num_txns += data.num_txns_;
+        total_num_bytes += data.num_bytes_;
+      }
+
+      EXPECT_EQ(total_num_txns, 101);
+      EXPECT_EQ(total_num_bytes, 91176);
```

Comment:
How did you get these values? My guess is trial and error, in which case this is probably not a good idea. `tested->SimulateOltp(100, 4)` is a **random** execution, and while we use a seed, I'm not convinced it will always succeed. Even when random testing, your tests should not fail even if you change the seed. Changing the seed would probably cause this to fail as varlen entries would be a different size, thus changing the number of byes.

I would do what `MetricsTest` does and just execute 1-3 predetermined transactions and test with those.

Reply:
After discussion we would just be testing whether the throughput is being logged or not, so this part can be left out.